source/ed/tests/fusioparser_test.cpp

```diff
@@ -416,4 +422,9 @@ BOOST_AUTO_TEST_CASE(sync_ntfs) {
     BOOST_CHECK_EQUAL(data.datasets[0]->realtime_level == nt::RTLevel::Base, true);
     BOOST_CHECK_EQUAL(data.datasets[0]->system, "obiti");
     BOOST_CHECK_EQUAL(data.vehicle_journeys[0]->dataset->uri, "d1");
+
+    const ed::types::VehicleJourney* vj = data.vehicle_journeys.front();
+    navitia::type::hasVehicleProperties has_vehicle_properties;
+    has_vehicle_properties.set_vehicle(navitia::type::hasVehicleProperties::BIKE_ACCEPTED);
+    BOOST_CHECK_EQUAL(vj->accessible(has_vehicle_properties.vehicles()), true);
```

Comment:
even if you did not change this, can you add a check in the docker tests ? this will ensure we write the field in the db and read it